check50/__main__.py

```diff
@@ -278,17 +284,25 @@ def main():
     global SLUG
     SLUG = args.slug
 
-
     if args.dev:
         args.offline = True
-        args.verbose = True
+        args.verbose = "info"
 
     if args.offline:
         args.local = True
 
-    if args.verbose:
-        # Show lib50 commands being run in verbose mode
-        logging.basicConfig(level=os.environ.get("CHECK50_LOGLEVEL", "INFO"))
+    if args.verbose == "info":
+        # Show lib50 commands being run
+        lib50_logger = logging.getLogger("lib50")
+        lib50_logger.setLevel(logging.INFO)
+        lib50_logger.addHandler(logging.StreamHandler(sys.stdout))
+        lib50.ProgressBar.DISABLED = True
+        args.log = True
+    elif args.verbose == "debug":
+        # Show lib50 commands being run and their output
+        lib50_logger = logging.getLogger("lib50")
+        lib50_logger.setLevel(logging.DEBUG)
+        lib50_logger.addHandler(logging.StreamHandler(sys.stdout))
```

Comment:
Should these lines be factored out into their own separate function to be able to reuse for `info` as well?

Reply:
Arguably a lot more from `main` should be factored out into separate functions... But I factored this out now ;)